src/Symfony/Component/Yaml/Inline.php

```diff
@@ -125,7 +125,7 @@ public static function parse($value, $flags = 0, $references = array())
      *
      * @throws DumpException When trying to dump PHP resource
      */
-    public static function dump($value, $flags = 0)
+    public static function dump($value, $flags = 0, $indent = 0)
```

Comment:
Adding this new argument will mess up with the BC layer for the old signature currently.